esp-hal/src/lcd_cam/lcd/i8080.rs

```diff
@@ -168,7 +164,7 @@ where
                 .lcd_byte_order()
                 .bit(false)
                 .lcd_2byte_en()
-                .bit(is_2byte_mode)
+                .bit(false)
```

Comment:
Is this a correct change or just a leftover?

Reply:
This is correct, it's overwritten later